pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -128,7 +128,7 @@ social:
   - tutorial[:name]                                 = hoc_s(prefix + "name")
   - tutorial[:shortdescription]                     = hoc_s(prefix + "shortdescription")
   - tutorial[:longdescription]                      = hoc_s(prefix + "longdescription")
-  - tutorial[:language]                             = hoc_s(prefix + "language")
+  - tutorial[:language]                             = hoc_language(tutorial[:languages_supported])
```

Comment:
I think this change also needs to be made for https://code.org/learn, [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/learn/index.haml#L150).

Reply:
In fact, production https://code.org/learn currently has blank language fields due to the strings being removed.  Making the same change there should fix it.

Reply:
Addressed this in [this PR](https://github.com/code-dot-org/code-dot-org/pull/43466) as well.